transformer/common_data_provider.go

```diff
@@ -111,12 +118,35 @@ func (c CommonDataProvider) getNodeName() (string, error) {
 	return nName, nil
 }
 
+// FetchServerVersion returns the version of the Kubernetes server
+func (c CommonDataProvider) FetchServerVersion() (version.Version, error) {
```

Comment:
Might be more comprehensive, wdyt?
```suggestion
func (c CommonDataProvider) FetchK8sVersion() (version.Version, error) {
```